source/renderer/app/components/wallet/paper-wallet-certificate/VerificationDialog.js

```diff
@@ -190,13 +189,20 @@ export default class VerificationDialog extends Component<Props, State> {
 
   resetForm = () => {
     const { form } = this;
-    form.$('password').reset();
-    form.$('recoveryPhrase').reset();
+    // Cancel all debounced field validations
+    form.each((field) => { field.debouncedValidation.cancel(); });
+    form.reset();
+    form.showErrors(false);
+
+    // Autocomplete has to be reset manually
+    // this.recoveryPhraseAutocomplete.clear();
```

Comment:
@DominikGuzei @tomothespian this is the same thing as my previous comment: https://github.com/input-output-hk/daedalus/pull/794/commits/f9667fb4a78ecba1101dc16d2b898a2399130cb3#r175191862